extension/content/components/recipes/form/RecipeFormHeader.js

```diff
@@ -71,6 +81,45 @@ export default function RecipeFormHeader() {
           Save
         </IconButton>
       </div>
+      <SaveModal
+        show={showCommentModal}
+        onClose={closeSaveModal}
```

Comment:
this is a confusing name since it is not an event handler but rather a method that is actually closing the modal. perhaps we should just pass through `setShowCommentModal` as a property named `setShowModal` and then it can decide whether to set `true` or `false` in the modal component itself.